packages/web-components/fast-foundation/src/flipper/flipper.ts

```diff
@@ -13,18 +13,18 @@ export { FlipperDirection };
  * @public
  */
 export type FlipperOptions = {
-    next?: string | SyntheticViewTemplate;
-    previous?: string | SyntheticViewTemplate;
+    nextIcon?: string | SyntheticViewTemplate;
+    previousIcon?: string | SyntheticViewTemplate;
 };
```

Comment:
I don't think we should change these - these may be text and not just icons.

Reply:
True, though it seems unlikely. I think the closest we have to this model is the breadcrumb item "separator", where we did not add "icon". Technically any "icon" or "indicator" content could just be text, but I do think in general having the hint at the intent is helpful. Following our convention, I think believe "content" is the only word we have that covers everything. I'm not sure how we should define the common rule for how we make this choice or distinction though.

Reply:
> I'm not sure how we should define the common rule for how we make this choice or distinction though.

I'm in agreement here which is why I favor making the correct choice per component rather than just broad consistency rules. I think this is a case where next/previous provides a better API than just enumerating that it's intended for an icon.